### PR TITLE
Add additional magnet scanner props

### PR DESCRIPTION
**Pull request by wilsonpage**
- Native code in the magnet-client now does more with scanner items meaning we needed to expose more properties.

### CODE REVIEW

**Comment by wilsonpage** [comment]
@arcturus r?

**Comment by wilsonpage** [comment]
Required for https://github.com/mozilla-magnet/magnet-client/pull/301

File: src/main/java/org/mozilla/magnet/scanner/MagnetScannerItem.java [hunk 1]
```diff
@@ -4,56 +4,105 @@
 import org.json.JSONObject;
 
 import java.util.HashMap;
+import java.util.Map;
 
 public class MagnetScannerItem {
     private double mDistance = -1;
+    private String mChannelId;
+    private String mImageUri;
```
Comment (arcturus) [inline src/main/java/org/mozilla/magnet/scanner/MagnetScannerItem.java:12]: I understand that we are adding this to try to save a call to the metadata service, but will it be enough? What if we want to add description too?
Should we just add the channel id and relay on a second call?

**Review by arcturus — APPROVED** [review]
After offline conversation those fields are for later use, which sounds pretty cool.

Shiiip it!

**Pull request merged**